tests/unit/services/event.bus.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { mocked } from 'ts-jest/utils'
+
+import EventBus from '@/services/EventBus'
+import RenderedIpc from '@/events/ipcs/RenderedIpc'
+import LauncherEvent from '@/events/LauncherEvent'
+import { DirectorySelected } from '@/events/ClientActions'
+
+jest.mock('@/events/ipcs/RenderedIpc')
+jest.mock('@/events/ClientActions')
+
+describe('Event Bus', function () {
```

Comment:
давай, коль везде стрелки - везде продолжать в одном стиле
```suggestion
describe('Event Bus', () => {
```